### PR TITLE
SPHP-1 Added queued tasks

### PR DESCRIPTION
**Pull request by beinarovic**

### CODE REVIEW

File: Api/Endpoints/Files.php [hunk 1]
```diff
@@ -68,6 +68,9 @@ public function fetchAll($projectId)
      */
     public function upload($projectId, $body)
     {
+        // Requests that do not use the queue=true parameter are deprecated and will be unsupported
+        $body['queue'] = true;
```
Comment (andrewLab-lokalise) [inline Api/Endpoints/Files.php:72]: Just double-check - is it really goes to body? I thought, it was query parameter

Comment (beinarovic) [reply Api/Endpoints/Files.php:72]: Ye, upload doesn't expect any specific query params.
https://app.lokalise.com/api2docs/curl/#resource-files

Comment (emsyzz) [inline Api/Endpoints/Files.php:72]: Forcing `queue` to be enabled my break stuff for clients. Didn't we have some transition period?

Comment (emsyzz) [reply Api/Endpoints/Files.php:72]: Or library is forcing queue since `3.0.0`?

Comment (beinarovic) [reply Api/Endpoints/Files.php:72]: This is a major version update (3.0.0). 2.* are deprecated, but you can use them. 3.0.0 is the version that works only in the new way (confirmed)

**Pull request merged**